cfpb/consumerfinance.gov
# P4C: scope guides CSS

The guides CSS in the P4C main.less file overrides our default table styles, which are now needed for the grad path tool. This PR scopes the guides CSS so it will only be used by the pages that need it.


## Changes

- Add parent wrapper class to guides pages and guides.less


## How to test this PR

1. Check out branch and run gulp
2. Navigate to [the grad path tool](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=110635&pid=4510-3&houp=onCampus&typp=bachelors&lenp=4&ratp=inState&depp=dependent&cobs=n&tuit=14184&hous=18222&book=849&subl=3000&houx=1395&fdx=498&clhx=127&trnx=750&hltx=265&entx=185&retx=282&taxx=143&othx=328) and open the `Review your financing plan` screen (under `Worth your investment`). Verify that tables display correctly.
3. Navigate to guides pages ([Manage your money](http://localhost:8000/paying-for-college/manage-your-college-money/), [Repay student debt](http://localhost:8000/paying-for-college/repay-student-debt/), [Choose a student loan](http://localhost:8000/paying-for-college/choose-a-student-loan/)) and verify that they display correctly


## Screenshots

### :(

<img width="508" alt="Screen Shot 2021-04-12 at 12 55 51 PM" src="https://user-images.githubusercontent.com/778171/114455471-a45baf00-9b90-11eb-971a-6c1e6b650353.png">



### :)

<img width="683" alt="Screen Shot 2021-04-12 at 12 55 32 PM" src="https://user-images.githubusercontent.com/778171/114455484-a7ef3600-9b90-11eb-9a55-068bf4034d4c.png">


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
